tests/base_test_class.py

```diff
@@ -476,7 +476,7 @@ def get_log(self, log_type):
 
                 # save the last timestamp only if newer
                 # in this set of logs
-                if entry["timestamp"] > last_timestamp:
+                if entry["timestamp"] > last_timestamp:  # noqa: PLR1730
```

Comment:
Why noqa here?

Reply:
wanted to resolve the other issue first.